supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog/www: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
supervisor: couldn't chdir to ~/Desktop/python/blog: ENOENT
supervisor: child process was not spawned
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
/usr/bin/python3: can't open file '~/Desktop/python/blog/www/app.py': [Errno 2] No such file or directory
Traceback (most recent call last):
  File "/home/jia/Desktop/python/blog/www/app.py", line 30, in <module>
    from orm import *
  File "/home/jia/Desktop/python/blog/www/orm.py", line 16, in <module>
    import aiomysql
ImportError: No module named 'aiomysql'
Traceback (most recent call last):
  File "/home/jia/Desktop/python/blog/www/app.py", line 30, in <module>
    from orm import *
  File "/home/jia/Desktop/python/blog/www/orm.py", line 16, in <module>
    import aiomysql
ImportError: No module named 'aiomysql'
Traceback (most recent call last):
  File "/home/jia/Desktop/python/blog/www/app.py", line 30, in <module>
    from orm import *
  File "/home/jia/Desktop/python/blog/www/orm.py", line 16, in <module>
    import aiomysql
ImportError: No module named 'aiomysql'
Traceback (most recent call last):
  File "/home/jia/Desktop/python/blog/www/app.py", line 30, in <module>
    from orm import *
  File "/home/jia/Desktop/python/blog/www/orm.py", line 16, in <module>
    import aiomysql
ImportError: No module named 'aiomysql'
Traceback (most recent call last):
  File "/home/jia/Desktop/python/blog/www/app.py", line 30, in <module>
    from orm import *
  File "/home/jia/Desktop/python/blog/www/orm.py", line 16, in <module>
    import aiomysql
ImportError: No module named 'aiomysql'
Traceback (most recent call last):
  File "/home/jia/Desktop/python/blog/www/app.py", line 30, in <module>
    from orm import *
  File "/home/jia/Desktop/python/blog/www/orm.py", line 16, in <module>
    import aiomysql
ImportError: No module named 'aiomysql'
Traceback (most recent call last):
  File "/home/jia/Desktop/python/blog/www/app.py", line 30, in <module>
    from orm import *
  File "/home/jia/Desktop/python/blog/www/orm.py", line 16, in <module>
    import aiomysql
ImportError: No module named 'aiomysql'
Traceback (most recent call last):
  File "/home/jia/Desktop/python/blog/www/app.py", line 30, in <module>
    from orm import *
  File "/home/jia/Desktop/python/blog/www/orm.py", line 16, in <module>
    import aiomysql
ImportError: No module named 'aiomysql'
